querybook/webapp/components/UserGroupCard/UserGroupCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from 'react';
+
+import { UserBadge } from 'components/UserBadge/UserBadge';
+import { IUserInfo } from 'const/user';
+import { UserResource } from 'resource/user';
+import { AccentText } from 'ui/StyledText/StyledText';
+
+import './UserGroupCard.scss';
+
+interface IProps {
+    userGroup: IUserInfo;
+}
+
+export const UserGroupCard = ({ userGroup }: IProps) => {
+    const [members, setMembers] = useState([]);
+
+    useEffect(() => {
+        UserResource.getUserGroupMembers(userGroup.id).then(({ data }) => {
+            setMembers(data);
+        });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    return (
+        <div className="UserGroupCard">
+            <div>
+                <AccentText color="dark" weight="bold">
+                    {userGroup.fullname}
+                </AccentText>
+                <AccentText color="light">{userGroup.username}</AccentText>
+            </div>
+            <div className="mt8">
+                <AccentText color="dark" weight="bold">
+                    Email
+                </AccentText>
+                <AccentText color="light">{userGroup.email}</AccentText>
+            </div>
+            <div className="mt8">
+                <AccentText color="dark" weight="bold">
+                    Description
+                </AccentText>
+                <AccentText color="light">
+                    {userGroup.properties?.description}
```

Comment:
can you document the structure of user properties for groups?

Reply:
I had this type info in `IUserInfo`, is that what you asking?

Reply:
hm im thinking to add a new wiki page that explains what info people can put into group's properties field.